samplesheets/tests/test_permissions_api.py

```diff
@@ -356,6 +362,141 @@ def test_get_archive(self):
         self.assert_response_api(url, self.anonymous, 401)
 
 
+class TestIrodsAccessTicketListAPIView(IrodsAccessTicketAPIViewTestBase):
+    """Test permissions for IrodsAccessTicketListAPIView"""
+
+    def setUp(self):
+        super().setUp()
+        self.url = (
+            reverse(
+                'samplesheets:api_irods_ticket_list',
+                kwargs={'project': self.project.sodar_uuid},
+            )
+            + '?active=0'
+        )
+        self.ticket = self.make_irods_ticket(
+            study=self.study,
+            assay=self.assay,
+            path=self.coll.path + '/ticket1',
```

Comment:
We should not have a iRODS collection in this class because we're not supposed to init these tests with iRODS and taskflow. You can just set up an imaginary path here, it will not be checked as we do not update the object in these tests.